rootex/framework/components/visual/spot_light_component.cpp

```diff
@@ -45,7 +45,7 @@ SpotLightComponent::SpotLightComponent(const float constAtt, const float linAtt,
     , m_DiffuseIntensity(diffuseIntensity)
     , m_Spot(spot)
 {
-	m_AngleRange = cos(angleRange * 3.1415 / 180);
+	m_AngleRange = angleRange;
```

Comment:
angle range has stopped working as shaders expect the cosine value to be there and this results in the number of degrees being sent to the shader